.php_cs

```diff
@@ -0,0 +1,59 @@
+<?php
+$header = <<<'EOF'
+
+ This file is part of the `liip/LiipImagineBundle` project.
+
+ (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+
+ For the full copyright and license information, please view the LICENSE.md
+ file that was distributed with this source code.
+EOF;
+
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        //'@PHP56Migration' => true,
```

Comment:
Can we get rid of any commented entries?

Reply:
ok, I leave it just in case that you want enable someone of them, that actually could be good for the project. I will remove all commented